src/pybind/mgr/dashboard_v2/tests/helper.py

```diff
@@ -24,12 +24,16 @@ class ControllerTestCase(unittest.TestCase):
     DASHBOARD_PORT = os.environ.get('DASHBOARD_V2_PORT', 8080)
 
     def __init__(self, *args, **kwargs):
+        self.dashboard_host = kwargs.pop('dashboard_host') \
```

Comment:
I don't think we need this for now. I understand you required this because of the iscsi test case, but as I commented previously you can avoid this.

Reply:
see above.